torch/fx/experimental/fx2trt/example/sparse_quantized_resnet_test.py

```diff
@@ -0,0 +1,316 @@
+from collections import defaultdict
+import copy
+import time
+
+import torch
+import torch.fx
+from torch.ao.quantization.quantize_fx import prepare_fx, convert_fx
+from torch.fx.experimental.fx2trt.fx2trt import TRTInterpreter, InputTensorSpec, TRTModule
```

Comment:
Nit: We have a recent refactor which changes some file structures.

```suggestion
from torch.fx.experimental.fx2trt import TRTInterpreter, InputTensorSpec, TRTModule
```